src/super_gradients/common/sg_loggers/wandb_sg_logger.py

```diff
@@ -102,7 +105,11 @@ def __init__(
                     )
                 wandb_id = self._get_wandb_id()
 
-        run = wandb.init(project=project_name, name=experiment_name, entity=entity, resume=resumed, id=wandb_id, **kwargs)
+        run = (
+            wandb.init(project=project_name, name=experiment_name, entity=entity, resume=resumed, id=wandb_id, sync_tensorboard=sync_tensorboard, **kwargs)
+            if wandb.run is None
```

Comment:
This is breaking the resume experiment feature.
Moreover, if the run was initialized from outside, we have no way of knowing it is logging into the correct experiment. 
Why would you want to init wandb from outside of the logger?

Reply:
Assume I trained my model fro 10 epochs and then stopped it for some reason. now I want to resume the training.
I can just use the `resume` flag in the training params and the wandb run will be connected to the right run ID.
but if I init the run from outside, how would I do this? I need to go to wandb, check the run Id, init the run with the correct run Id and the resume flag. If I just call wandb.init() without the id, the run will not resume.